src/SmartComponents/Group/Groups.js

```diff
@@ -45,6 +45,12 @@ class Groups extends Component {
       scrollToTop();
     }
 
+    componentDidUpdate(prevProps) {
+      if (this.props.groups !== prevProps.groups) {
```

Comment:
This is gonna trigger every time anything happens with props. in JS `[1,1] !== [1,1]`. Props are always immutable objects and arrays will be different instances so this its always `true`. Use `lodash.isEqual` to compare arrays or objects.